README.md

```diff
@@ -6,125 +6,160 @@
+function was invoked.
 
-In JavaScript, a similar concept applies, but instead of using `self`, we use the keyword `this`. The biggest difference between `this` and `self` is that `self` always refers to an object, but in JavaScript, `this` can refer to whatever called the function: an event, action, or object that called the function.
-Every new function gets its own `this` value.
+## Function (Local) Scope
 
-There are four main ways to define the value of `this`, which is determined by the execution context of the function call.
+Inside of a function, the value of `this` will be set based on how the
+function is called.
 
-four rules and order of precedence:
-+ default binding - in strict mode - default to undefined, default to global object
+### Simple Function Call
 
-## Global Scope
-
-If we're referencing `this` in the global scope (outside of a function), it can refer to `window` or `document`.
+Add the following to `script.js` and refresh the page with the console
+open:
 
 ```js
-this === window;
-// returns true
-
-this === document;
-// returns true
-
-function myFunk(){
-  console.log(this + " is funky");
+function checkThis(){
+  console.log(this);
 }
-myFunk();
-// prints out "[object Window] is funky" because the function is called on the window object.
+checkThis();
+// outputs Window object
 ```
-### Strict Mode
 
-If your function is in strict mode (a mode which allows for better error checking by prohibiting the use of implicitly declared variables, duplicate parameter names, and other potentially bug-causing behavior), then the value of `this` becomes `undefined` instead of `window`:
+This will also output the `Window` object, because this simple function
+call doesn't set the `this` value, and because we aren't in *strict
+mode*, the value of `this` must be an object, so the default global
+object `Window` is used.
+
+**Advanced:** Strict mode is a setting that enables better error-checking in your code
+by prohibiting the use of implicitly declared variables, duplicate
+parameter names, and other potentially bug-causing behavior, and
+converting some silent execution errors into not-so-silent ones. For
+more information, check out the [MDN documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode).
+
+Let's put our function in strict mode:
 
 ```js
-function sayHi(){
+function checkThis(){
   'use strict';
-  alert("Hey!! " + this);
+  console.log(this);
 }
-//alerts "Hey!! undefined"
+checkThis();
+// outputs "undefined"
 ```
 
-## Function Scope
+In strict mode, `this` remains at whatever it's set to when entering the execution context. If it's not defined, it remains undefined rather than being assigned the default `Window` object.
+
+### Constructor Function Call
 
-Let's look at a more detailed example. Let's define a function that prints `'Hello, ${name}!'`. It accepts an object that has a `name` property.
+Remember the use of `this` in our constructor functions?
```

Comment:
So one thing that we're trying to do is "decouple" a lot of our curriculum. Is there a way to word this so that it reminds students about using `this` in constructor functions but doesn't point to previous lessons?